dbActions.py

```diff
@@ -0,0 +1,71 @@
+import sqlite3
+import Settings
```

Comment:
[Modules should have short, all-lowercase names.](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)